scripts/screenshots.js

```diff
@@ -0,0 +1,140 @@
+const puppeteer = require('puppeteer')
+const path = require('path')
+const fs = require('fs')
+const sortBy = require('lodash/sortBy')
+
+const DEFAULT_PUPPETEER_VIEWPORT = {
+  width: 800,
+  height: 600
+}
```

Comment:
Are we sure to want this resolution as default? We should use the most used resolution on our product

Reply:
This is the default resolution of puppeeter so that we can reset the viewport to the original size. Then, for each component the viewport is scaled to accomodate the full height of the examples.

Reply:
We could set the viewport to the most used resolution or another resolution but this was not what this code was meant to do.